### DISCUSSION TITLE
Create a more splitted territory to increase LCZ accuracy for urban types

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#836] [Ideas]
It seems that due to our way to create the TSU, we miss some quite large residential areas (all the small bourgs along roads) when calculating the LCZ. A good example here in the South of Nantes:
![image](https://github.com/orbisgis/geoclimate/assets/13120405/81f828e5-e906-4320-a7b6-2c78de6877d2)
![image](https://github.com/orbisgis/geoclimate/assets/13120405/2f845ab2-c0ee-425e-a9c0-c9950994361e)
--> a good way to deal with that would be to use the residential / industrial / commercial / etc. zones to partition the territory

**Comment by j3r3m1** [comment]
Same as in #834

**Comment by ebocher** [comment]
What about this  ? ;-)

![urban_areas](https://github.com/orbisgis/geoclimate/assets/1820572/e2d31259-685d-4c78-8413-7ab1df6450b8)
